### ISSUE TITLE
Change Design of Comparison Page and NavBar

### ISSUE THREAD

**Issue by monasobh** [hyrise/Cockpit#691]
**Is your feature request related to a problem? Please describe:**  
The Navigationbar is taking away too much space, the focus should be on the metrics. 
The metrics design should be similar to the overview page and have a little margin. 

**Please describe the solution you'd like:**  
Make Navigationbar smaller, add margin between metrics and make metrics header smaller. 

**Affected Component(s):**  
`ComparisonPage`, `UI`,

Before: 
<img width="640" alt="Screenshot 2020-07-14 at 19 18 24" src="https://user-images.githubusercontent.com/49497676/87531326-d38cd080-c691-11ea-867e-eb37357809cc.png">

Changes: 
- rename "Analysis Views" --> Monitoring
- rename "Settings" --> Configuration
- change size of Navbar
- changed color of "Aggregation: 1 second" to our primary color (Darkgrey)
- reduced size of graphs (remove white space between headline and metric) 
- added space between graphs 
- make y-axis font smaller 
- added light-grey background 
- changed font size of metric (similar to the db name in db information)

After: 
<img width="640" alt="Screenshot 2020-07-17 at 15 38 20" src="https://user-images.githubusercontent.com/49497676/87794159-869e2b00-c846-11ea-8278-032fe597f48b.png">
-----

Changes (from left to right): 
<img width="342" alt="Screenshot 2020-07-17 at 23 38 22" src="https://user-images.githubusercontent.com/49497676/87832604-a8b79d80-c886-11ea-9d8d-41cf754102e6.png"><img width="325" alt="Screenshot 2020-07-18 at 01 15 26" src="https://user-images.githubusercontent.com/49497676/87837552-3352c980-c894-11ea-8bab-3015ff82c836.png">

**Comment by Bensk1** [comment]
Looks good to me, thanks!

> rename "Analysis Views" --> Monitoring

Was that someone's request?

**Comment by monasobh** [comment]
No, just my suggestion. It's shorter and often used in other database dashboards.

**Comment by Bouncner** [comment]
> We use different size and style of the font-family Roboto everywhere.

I just saw the change to Roboto without providing alternatives fonts. What happens on a computer without Roboto? I just remember that we had the custom font discussion before and I was surprised to see that coming up again.

**Comment by monasobh** [comment]
Roboto is a Google Font which means that the browsers (Chrome, Firefox, Safari, Opera, Internet Explorer) are supported by those fonts. Using an unsupported browser means that the text is displayed in the unsupported browser's default font.

**Comment by Bouncner** [comment]
That’s assuming users allow font downloading (should be true for most, but not for all).
And I assume that this material framework has better fall backs than the default and rather some other serifless fonts.
What is bothering me is that you haven’t provided these. What happens without Roboto?

**Comment by monasobh** [comment]
So you would like us to set a second font in our code in case Roboto is not being supported by the browser? Do you know what font that would be?

**Comment by Bouncner** [comment]
I'd say `font-family: Sans-Serif;` but I don't really have a plan. :D

**Comment by monasobh** [comment]
Unfortunately it's not possible to just set the font-family to sans-serif. I would say that if someone denied the font downloading in their browser, it's probably on purpose (because it's activated by default). For me, it would not make a lot of sense to set another fall back font, unknowingly if that font even exists in the browser, where the font-downloading is off.

**Comment by Bouncner** [comment]
Why shouldn’t this be possible?
From my understanding it’s best practice: https://www.w3schools.com/cssref/css_websafe_fonts.asp

Relying on custom fonts (still, without even having a good reason to use them in the first place IMO) is just wrong.

**Comment by schTi** [comment]
Our material design framework vuetify already sets `sans-serif` as fallback font

**Comment by Bouncner** [comment]
Good to know, but if we use Roboto anyway and have a fall-back, why set it in the discussed PR?

**Comment by schTi** [comment]
> Good to know, but if we use Roboto anyway and have a fall-back, why set it in the discussed PR?

Ah sorry nvm, I just noticed this PR adds Roboto for all plotly charts. So then this has nothing to do with vuetify. @monasobh Just do this: https://plotly.com/javascript/reference/#layout-font-family

**Comment by monasobh** [comment]
> Good to know, but if we use Roboto anyway and have a fall-back, why set it in the discussed PR?

In this PR I just changed the description of the y-axis of the metrics to the same font we (Vuetify) use which is Roboto. Because we use plotly for our metrics we have to separately set the font for the x- and y-axis to Roboto to be consistent, because the default font of plotly is not Roboto. 

@schTi I already added "Roboto, regular" but I might as well change it to "Roboto, sans-serif".
